modules/k8s/cluster_role.go

```diff
@@ -0,0 +1,23 @@
+package k8s
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// GetClusterRole returns a Kubernetes ClusterRole resource with the given name. This will fail the test if there is an error.
+func GetClusterRole(t *testing.T, options *KubectlOptions, roleName string) *rbacv1.ClusterRole {
+	role, err := GetClusterRoleE(t, options, roleName)
+	require.NoError(t, err)
+	return role
+}
+
+// GetClusterRoleE returns a Kubernetes ClusterRole resource with the given name.
+func GetClusterRoleE(t *testing.T, options *KubectlOptions, roleName string) (*rbacv1.ClusterRole, error) {
+	clientset, err := GetKubernetesClientFromOptionsE(t, options)
+	require.NoError(t, err)
```

Comment:
Sorry, I missed this the first go around. The `E` variant should not have any `require` or `t.Fatal` calls and return the error. So this should be:

```
if err != nil {
    return nil, err
}
```

Reply:
ah, of course ... sorry. Next time i'll be prepared better :D